src/components/Button/ButtonAddPostTopic.js

```diff
@@ -35,6 +36,10 @@ export default ButtonAddPostTopic;
 
 const styles = StyleSheet.create({
   container: {
+    position: 'absolute',
+    bottom: normalize(30),
+    right: 0,
+    zIndex: 99,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `zIndex` property is used to control stack order of elements. However, it's not always guaranteed to work as expected in React Native, especially when dealing with Android due to certain limitations. If you encounter any issues related to stacking or overlaying components, consider using other methods such as reordering components or using a modal for floating buttons.
```diff
  container: {
    position: 'absolute',
    bottom: normalize(30),
    right: 0,
-   zIndex: 99,
    backgroundColor: '#0000'
  },
```
Please test thoroughly on different devices and platforms to ensure the desired behavior.


<!-- This is an auto-generated comment by OSS CodeRabbit -->